config/environments/development.rb

```diff
@@ -4,6 +4,18 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
+
+  config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: 587,
+    domain: "example.com",
+    authentication: "plain",
+    enable_starttls_auto: true,
+    user_name: ENV["GMAIL_USERNAME"],
+    password:ENV["GMAIL_PASSWORD"]
+  }
+config.action_mailer.default_url_options = { :host => 'localhost:3000' }
```

Comment:
We'll want to tab this one over once, and we want to use the front-end, so port 5000.